src/api/v1beta1/feature_flags_resources.go

```diff
@@ -5,61 +5,6 @@ import (
 	"k8s.io/apimachinery/pkg/api/resource"
 )
 
-const (
-	annotationFeatureActiveGate = AnnotationFeaturePrefix + "activegate-"
-	annotationFeatureEec        = annotationFeatureActiveGate + "eec-"
-	annotationFeatureStatsd     = annotationFeatureActiveGate + "statsd-"
-)
-
-// FeatureEecResourcesRequests is a feature flag to define CPU or memory requests for the EEC container
-func (dk *DynaKube) FeatureEecResourcesRequests(resourceName corev1.ResourceName) *resource.Quantity {
-	return eecResourceRequirements(dk, "requests-"+resourceName)
-}
-
-// FeatureEecResourcesLimits is a feature flag to define CPU or memory limits for the EEC container
-func (dk *DynaKube) FeatureEecResourcesLimits(resourceName corev1.ResourceName) *resource.Quantity {
-	return eecResourceRequirements(dk, "limits-"+resourceName)
-}
-
-// FeatureStatsdResourcesRequests is a feature flag to define CPU or memory requests for the StatsD container
-func (dk *DynaKube) FeatureStatsdResourcesRequests(resourceName corev1.ResourceName) *resource.Quantity {
-	return statsdResourceRequirements(dk, "requests-"+resourceName)
-}
-
-// FeatureStatsdResourcesLimits is a feature flag to define CPU or memory limits for the StatsD container
-func (dk *DynaKube) FeatureStatsdResourcesLimits(resourceName corev1.ResourceName) *resource.Quantity {
-	return statsdResourceRequirements(dk, "limits-"+resourceName)
-}
-
-func formatResourceName(resourceName corev1.ResourceName) string {
-	return "resources-" + string(resourceName)
-}
-
-func eecResourceRequirements(dk *DynaKube, resourceName corev1.ResourceName) *resource.Quantity {
-	return resourceRequirements(dk, annotationFeatureEec, resourceName)
-}
-
-func statsdResourceRequirements(dk *DynaKube, resourceName corev1.ResourceName) *resource.Quantity {
-	return resourceRequirements(dk, annotationFeatureStatsd, resourceName)
-}
-
-func resourceRequirements(dk *DynaKube, flagPrefix string, resourceName corev1.ResourceName) *resource.Quantity {
-	flagName := flagPrefix + formatResourceName(resourceName)
-
-	val, ok := dk.Annotations[flagName]
-	if !ok {
-		return nil
-	}
-
-	quantity, err := resource.ParseQuantity(val)
-	if err != nil {
-		log.Info("Problem parsing resource requirements for", "flagName", flagName, "val", val, "err", err.Error())
-		return nil
-	}
-
-	return &quantity
-}
-
 // +kubebuilder:object:generate=false
 type ResourceRequirementer interface {
 	Requests(corev1.ResourceName) *resource.Quantity
```

Comment:
none of these are needed, they are only used in the tests file for this file :)

Reply:
so the whole `src/api/v1beta1/feature_flags_resources.go` can be removed (with its test)